### PR TITLE
🌱 Release v1.4.7

### PR DESCRIPTION
**Pull request by g-gaston**
/area release

### CODE REVIEW

File: CHANGELOG/v1.4.7.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.23.x -> v1.27.x
+- Workload Cluster: v1.21.x -> v1.27.
```
Comment (killianmuldoon) [inline CHANGELOG/v1.4.7.md:4]: nit
```suggestion
- Workload Cluster: v1.21.x -> v1.27.x
```

File: CHANGELOG/v1.4.7.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Changes since v1.4.6
+---
+## :chart_with_upwards_trend: Overview
+- 10 new commits merged
+- 2 bugs fixed 🐛
+
+## :bug: Bug Fixes
+- KCP: Fix KCP Controller reconcile always return error when workload cluster is unreachable (#9450)
+- machinehealthcheck: Fix excessive trace logging in the machine health check controller (#9427)
```
Comment (killianmuldoon) [inline CHANGELOG/v1.4.7.md:16]:
```suggestion
- MachineHealthCheck: Fix excessive trace logging in the machine health check controller (#9427)
```

probably could be fixed up in the notes tool

**Comment by killianmuldoon** [comment]
The verifier job isn't blocking so it looks  like we'll have to add another valid emoji to get this merged. I think this should still be handled by the fallback option from https://github.com/kubernetes-sigs/cluster-api/pull/9444 for now so it's not a major issue.

We might have to do this over the course of the next few releases until there's a change in the kubebuilder release tools - [is someone planning on opening an issue over at that repo](https://github.com/kubernetes-sigs/kubebuilder-release-tools)? 

Alternatively we could use something like `:sparkles: [Release] v1.4.7` and ignore PRs based off the [Release] tag.

**Review by furkatgofurov7 — APPROVED** [review]
/approve

**Pull request merged**